examples/notebooks/parameter-management.ipynb

```diff
@@ -0,0 +1,373 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parameter management\n",
+    "\n",
+    "PyBamm comes with a set of pre-defined parameters for common chemistries. These files are located inside your PyBaMM installation directory, typically something like\n",
+    "```\n",
+    "../venv/lib/pythonX.Y/site-packages/pybamm/input/parameters\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `input/parametes` directory is organised as follows:\n",
+    "```\n",
+    "input/parameters\n",
+    "  lithium-ion/ # chemistry\n",
+    "    anodes/ # Component\n",
+    "      graphite_Chen2020/ # parameter set\n",
+    "        parameters.csv\n",
+    "\tgraphite_LGM50_diffusivity_Chen2020.py\n",
+    "        ...\n",
+    "      graphite_mcmb2528_Marquis2019/\n",
+    "      graphite_Ecker2015/\n",
+    "      ...\n",
+    "    cathodes/\n",
+    "    cells/\n",
+    "    electrolytes/\n",
+    "    seis/\n",
+    "    separators/\n",
+    "    experiments/\n",
+    "  lithium-acid/\n",
+    "    ...\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When setting parameter values using `pybamm.ParameterValues` and a filename, the file is searched in several locations, described by\n",
+    "the variable `pybamm.PARAMETER_PATH`.\n",
+    "The default is "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['/tmp',\n",
+       " '/tmp/virtualenvs/pybamm-venv/lib/python3.7/site-packages/pybamm/input/parameters']"
```

Comment:
I get a different output:
```
['/Users/vsulzer/Documents/Energy_storage/PyBaMM/examples/notebooks',
 '/Users/vsulzer/Documents/Energy_storage/PyBaMM/pybamm/input/parameters']
```

Reply:
That's expected: the should be
```
['/the/current/directory',
 '/path/to/pybamm/input/parameters']
```
You run the notebook in `/Users/vsulzer/Documents/Energy_storage/PyBaMM/examples/notebooks`, and probably installed pybamm in editable mode so default parameters are located in `/Users/vsulzer/Documents/Energy_storage/PyBaMM/pybamm/input/parameters`

Reply:
Actually you're right, that was confusing. I modified default output of the notebook to display my personal pybamm install instead of `/tmp`